qrexec/tests/qrexec_policy_daemon.py

```diff
@@ -181,3 +218,208 @@ async def test_wrong_arg(self, mock_request, async_server, tmp_path):
         await self.send_data(async_server, tmp_path, data)
 
         mock_request.assert_not_called()
+
+    @pytest.mark.asyncio
+    async def test_simple_qrexec_request(
+            self, mock_request, qrexec_server, tmp_path, mock_system):
+
+        for i in server_types:
+            data = b'policy.Eval%s+d c keyword @adminvm\0a\0b' % i
+
+            assert await self.send_data(qrexec_server[i], tmp_path, data, i) == b'result=deny\n'
+
+            mock_request.assert_called_once_with(
+                domain_id='dummy_id', source='a', intended_target='b',
+                service_and_arg='d', process_ident='0', log=log,
+                assume_yes_for_ask=True, just_evaluate=True,
+                policy_cache=unittest.mock.ANY)
+            mock_request.reset_mock()
+
+    @pytest.mark.asyncio
+    async def test_not_guivm(
+            self, mock_request, qrexec_server, tmp_path, mock_system):
+        data = b'policy.EvalGUI+d b keyword @adminvm\0c\0a'
+        assert len(qrexec_server) == 2
```

Comment:
Test for testing tests? ;) This would make adding tests for other sockets in the future annoying. Similar in the tests below.